cmd/access/node_builder/access_node_builder.go

```diff
@@ -1020,6 +1045,10 @@ func (builder *FlowAccessNodeBuilder) extraFlags() {
 			"execution-data-sync-enabled",
 			defaultConfig.executionDataSyncEnabled,
 			"whether to enable the execution data sync protocol")
+		flags.BoolVar(&builder.publicNetworkExecutionDataEnabled,
+			"public-network-execution-data-sync-enabled",
+			defaultConfig.publicNetworkExecutionDataEnabled,
+			"whether to enable the execution data sync protocol on public network")
```

Comment:
let's make it clear this is not ready for production use
```suggestion
			"[experimental] whether to enable the execution data sync protocol on public network")
```